brainkeeper/package.json

```diff
@@ -20,9 +20,12 @@
     "@angular/material": "^8.2.3",
     "@angular/platform-browser": "~8.2.9",
     "@angular/platform-browser-dynamic": "~8.2.9",
-
     "@angular/router": "^8.2.14",
+    "blueimp-canvas-to-blob": "^3.17.0",
+    "compress.js": "^1.1.2",
```

Comment:
Do you use these libraries? If not please remove them.

Reply:
No i don't use them.